pg_backup_api/README.md

```diff
@@ -0,0 +1,46 @@
+# pg-backup-api
+A server that provides an HTTP API to interact with Postgres backups
+
+
+# To run the Flask app
+
+Install `pg_backup_api` and change to the Barman user, then run
+
+`pg-backup-api serve`
+
+
+# Code Autogeneration
+
+This only needs to be done if the spec file has been changed.
+
+To generate new server controller/model code from the spec,
+
+- Make sure you have `openapi-generator` and `black` installed on your computer
+
+- Set the env variable `PYTHON_POST_PROCESS_FILE` to where the `black` executable is installed
+
+- From the top-most `pg_backup_api` directory, run
+`openapi-generator generate --enable-post-process-file -i pg_backup_api/spec/pg_backup_api.yaml -o pg_backup_api -t ../generator_templates -g python-flask`
+
```

Comment:
Weird. I'm also running Python 3.9. Maybe it's the version of black? I'm using version 21.9b0. 

I'm also wondering, once we get black set up as a GitHub Action like we have on the barman repo, maybe it's sort of moot? since presumably that would reformat everything to one central standard at PR/merge time.